cfgov/cfgov/settings/base.py

```diff
@@ -398,28 +402,52 @@
 
 PRIVACY_EMAIL_TARGET = os.environ.get("PRIVACY_EMAIL_TARGET", "test@localhost")
 
-
 # Password Policies
-# cfpb_common password rules
-CFPB_COMMON_PASSWORD_RULES = [
-    [r".{12,}", "Minimum allowed length is 12 characters"],
-    [r"[A-Z]", "Password must include at least one capital letter"],
-    [r"[a-z]", "Password must include at least one lowercase letter"],
-    [r"[0-9]", "Password must include at least one digit"],
-    [
-        r"[@#$%&!]",
-        "Password must include at least one special character (@#$%&!)",
-    ],
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
+        "OPTIONS": {
+            "min_length": 12,
+        },
+    },
+    {
+        "NAME": "login.password_validation.ComplexityValidator",
```

Comment:
I notice that when Wagtail displays the list of password rules, having these all combined into a single validator makes them look different from the other rules:

<img width="1136" alt="image" src="https://user-images.githubusercontent.com/654645/231569056-baf5f9be-98cd-4d0d-b084-b1c0938a10bd.png">

Maybe this is just a minor annoyance not worth fixing, but it occurs to me that we could instead define 3 RegexValidators (or something similar) instead of one combined one like this.

Reply:
> we could instead define 3 RegexValidators (or something similar) instead of one combined one like this.

Unfortunately it looks like Django's password validators are a special, separate class of validators, and `RegexValidator` by itself won't work. This configuration:

```python
AUTH_PASSWORD_VALIDATORS = [
    {
        "NAME": "django.core.validators.RegexValidator",
        "OPTIONS": {
            "regex": r"[A-Z]",
            "message": "Password must include at least one capital letter",
        }
    },
]
```

Results in this error when trying to load the password reset page:

```
AttributeError: 'RegexValidator' object has no attribute 'get_help_text'
```

We could do a light subclass of `RegexValidator` for passwords and add the `get_help_text` method.

Reply:
Done in [dd3709a](https://github.com/cfpb/consumerfinance.gov/pull/7650/commits/dd3709a50904961ebbc75e7b7bc7986ad4008133)

Reply:
And it looks much better!

<img width="423" alt="image" src="https://user-images.githubusercontent.com/10562538/231792240-4881018e-57e4-4a67-a87d-76df227136fb.png">